pipeline/scripts/ci/manage_reports.py

```diff
@@ -0,0 +1,137 @@
+
+usage = """
+This script helps to update reports in IBM.
+
+Usage:
+  manage_reports.py (upload|download) <bucket_name> (-f FILE_PATH) (-k OBJ_KEYNAME)
+  manage_reports.py delete_object <bucket_name> (-k OBJ_KEYNAME)
+  manage_reports.py list_objects <bucket_name>
+  manage_reports.py -h | --help
+
+    action              upload | download | delete | list-objects
+    bucket_name         bucket name
+
+Example:
+    python manage_reports.py upload qe-ci-reports test.xml test-run-1
+    python manage_reports.py download qe-ci-reports test.xml test-run-1
+    python manage_reports.py delete qe-ci-reports test-run-1
+    python manage_reports.py list_objects qe-ci-reports
+
+Options:
+    -f <file> --file_path <file>    ex., /tmp/test.xml
+    -k <str> --object-key <str>     store file content as object with reference key name
+    -h --help                       Show this screen
+
+"""
+cos = CloudObjectStorage()
+
+
+def upload_objfile(bucket: str, local_file: str, obj_key: str):
+    """Upload file as file object
+
+    Args:
+        bucket: bucket name
+        local_file: local file path
+        obj_key: object name to store file
+
+    """
+    with open(local_file, "rb") as in_file:
+        cos.resource.Bucket(bucket).upload_fileobj(in_file, obj_key)
+    LOG.info(f"Uploaded successfully object({obj_key}) using file({local_file})")
+
+
+def download_objfile(bucket: str, local_file: str, obj_key: str):
+    """Upload file as file object
+
+    Args:
+        bucket: bucket name
+        local_file: local file path
+        obj_key: object name to store file
+
+    """
+    with open(local_file, "wb") as out_file:
+        cos.resource.Bucket(bucket).download_fileobj(obj_key, out_file)
+    LOG.info(
+        f"Downloaded successfully using KeyName({obj_key}) into file({local_file})"
+    )
+
+
+def delete_objfile(bucket: str, obj_key: str):
+    """delete file object
+
+    Args:
+        bucket: bucket name
+        obj_key: object name to store file
+    """
+    obj = cos.resource.Bucket(bucket).Object(obj_key)
+    obj.delete()
+    obj.wait_until_not_exists()
+    LOG.info(f"Deleted successfully the file object({obj_key})")
+
+
+def list_objects(bucket: str):
+    """list file object from bucket
+
+    Args:
+        bucket: bucket name
+    Returns:
+        list
+    """
+    _objs = cos.client.list_objects(Bucket=bucket)
+    print([i["Key"] for i in _objs.get("Contents")])
+
+
+OPS = {
+    "upload": upload_objfile,
+    "download": download_objfile,
+    "delete_object": delete_objfile,
+    "list_objects": list_objects,
+}
+
+if __name__ == "__main__":
+
+    _args = docopt(usage, help=True, version=None, options_first=False)
+    logging.basicConfig(
+        handlers=[logging.StreamHandler(sys.stdout)],
+        level=logging.INFO,
+        format="%(asctime)s - %(levelname)s - %(filename)s:%(lineno)d - %(message)s",
+    )
+
+    try:
+        _def = None
+        for key, value in OPS.items():
+            if _args[key]:
```

Comment:
this would result in attribute error for an incorrect value.

Reply:
Thanks.